common.go

```diff
@@ -134,13 +134,18 @@ func (m metricMap) writeToGoogleCloudMonitoring(keys []familyKey, labels map[str
 	if err != nil {
 		log.Fatalf("Cannot connect to GCP monitoring: %v\n", err)
 	}
+	errorCount := 0
 	for _, v := range m {
 		err := g.writeTimeSeriesValue(v, labels, timestamp)
 		if err != nil {
-			log.Fatal(errors.Wrap(err, "error writing metric: "+v.name))
+			log.Println(errors.Wrap(err, "error writing metric: "+v.name))
+			errorCount++
 		}
 		fmt.Print(".")
 	}
+	if len(m) < 20*errorCount {
```

Comment:
I don't think this is 5% or am I dumb? (both may be true)
```suggestion
	if errorCount > len(m)*0.05 {
```

Reply:
This is a way to calculate 5% to avoid division/multiplying float numbers.

Divide both sides by `20 * len(m)` and you'll get:
```
0.05 < errorCount / len(m)
```
Since this is the correct expression, the original is correct as well.

Reply:
Ah yeah.